packages/frames.js/src/middleware/farcaster.ts

```diff
@@ -76,16 +76,20 @@ export function farcaster(): FramesMiddleware<any, FramesMessageContext> {
       return next();
     }
 
-    const message = await getFrameMessage(payload, {
-      fetchHubContext: false,
-    });
+    try {
+      const message = await getFrameMessage(payload, {
+        fetchHubContext: false,
+      });
 
-    return next({
-      message,
-      clientProtocol: {
-        id: "farcaster",
-        version: "vNext",
-      },
-    });
+      return next({
+        message,
+        clientProtocol: {
+          id: "farcaster",
+          version: "vNext",
+        },
+      });
+    } catch (error) {
```

Comment:
isn't it also good to warn here in console, something like `farcaster: unable to parse message, moving to next middleware`?